src/backend/joanie/core/utils/contract.py

```diff
@@ -149,3 +151,21 @@ def generate_zip_archive(pdf_bytes_list: list, user_uuid: str, zip_uuid=None) ->
     storage.save(name=zip_archive_name, content=zip_buffer)
 
     return zip_archive_name
+
+
+def organization_has_owner(order_uuid=None) -> bool:
```

Comment:
Inject the full order here, not the id. In the caller you already have the order so there is no need to pass the id to find again the order.

Reply:
I like this idea, since its the Order object that initializes the whole procedure by submitting for signature the contract.